packages/cli/test/controller.js

```diff
@@ -25,80 +30,206 @@ describe('generator-loopback4:controller', tests);
 
 describe('lb4 controller', () => {
   it('does not run without package.json', () => {
-    helpers
-      .run(generator)
-      .withPrompts(withInputProps)
-      .then(() => {
-        assert.noFile(withInputName);
-      });
+    return runGenerator(true, true).then(() => {
+      assert.noFile(withInputName);
+    });
   });
   it('does not run without the loopback keyword', () => {
-    let tmpDir;
-    helpers
-      .run(generator)
-      .inTmpDir(dir => {
-        tmpDir = dir;
-        fs.writeFileSync(
-          path.join(tmpDir, 'package.json'),
-          JSON.stringify({
-            keywords: ['foobar'],
+    return runGenerator(true, true).then(() => {
```

Comment:
Huh, what does `true, true` mean and how does it differ from `false, true`? Boolean arguments are evil, please used an `options` object with named properties.

```
runGenerator({useTempDir: true, withPrompts: true})
```

Reply:
I knew I should've gone with my gut instinct!